networking/network_observability/network-observability-overview.adoc

```diff
@@ -32,23 +32,28 @@ The Network Observability Operator provides the Flow Collector API custom resour
 {product-title} console integration offers overview, topology view and traffic flow tables.
 
 [id="network-observability-dashboards"]
-=== Network Observability metrics
+=== Network Observability metrics dashboards
 
-The {product-title} console offers the *Overview* tab which displays the overall aggregated metrics of the network traffic flow on the cluster. The information can be displayed by node, namespace, owner, pod, and service. Filters and display options can further refine the metrics.
+The {product-title} console offers the *Overview* tab which displays the overall aggregated metrics of the network traffic flow on the cluster. The information can be displayed by node, namespace, owner, pod, and service. Filters and display options can further refine the metrics. Infrastructure and Application metrics are shown in a split-view for namespace, pod, and workloads. 
```

Comment:
```suggestion
On the *Overview* tab in the {product-title} console, you can view the overall aggregated metrics of the network traffic flow on the cluster. You can choose to display the information by node, namespace, owner, pod, and service. You can further refine the metrics by using filters and display options. Namespace, pod, and workloads for Infrastructure and Application metrics display in a split-view. 
```
- Rephrase the sentences to lead with the main subject of the content (the **Overview** tab), to make the sentences more user-centric, and to avoid passive voice. My suggested edits show one possible way to achieve these goals.
- Are `Infrastructure` and `Application` metrics UI elements? If so, format the terms as Bold. If not, why are these terms capitalized?

Reply:
Yeah they are in the UI like this:
![image](https://github.com/openshift/openshift-docs/assets/77501157/89a1d74a-2374-48ca-a904-edb8fd577fe1)